### PR TITLE
feat: [SSCA-66] ssca artifact list view | using restful react temporarily | start integrating react-ssca-service-client

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary
image

TODO: 
- add tests with a different ticket/ PR
- move to open API react SDK once ingress rules and paths are fixed as per https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/21257650399/Injecting+microservice+details+in+new+APIs+Reverse-Proxy#2.-Ingress-Simplification

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: configs/devServerProxy.config.js [hunk 1]
```diff
@@ -192,12 +192,11 @@ module.exports = {
...
     target: process.env.IAC_UI_URL || 'https://localhost:8185'
   },
-  '/ssca': {
-    pathRewrite: { '^/ssca': '' },
-    target: process.env.SSCA_UI_URL || 'https://localhost:8186'
-  },
   '/ssca/api': {
-    pathRewrite: { '^/ssca/api': '/api' },
     target: process.env.SSCA_API_URL || 'https://localhost:8186'
+  },
```
Comment (udayvunnam) [inline configs/devServerProxy.config.js:197]: Will get rid of this once open API sdk is integrated for ssca

File: src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/PolicyViolations/PolicyViolationsDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        icon="cross"
+        withoutBoxShadow
+        onClick={() => showEnforcementViolations()}
+      />
+      {loading ? (
+        <PageSpinner />
+      ) : data ? (
+        <>
+          <Heading level={2} color={Color.GREY_800} font={{ weight: 'bold' }} padding="large">
+            Artifact Violation Details
```
Comment (PratyushGar) [inline src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/PolicyViolations/PolicyViolationsDrawer.tsx:72]: Please move this to yaml

**Review by PratyushGar — APPROVED** [review]

File: src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/PolicyViolations/PolicyViolationsDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              autoFocus={false}
+            />
+          </div>
+
+          <PolicyViolationsTable data={data} />
+        </>
+      ) : (
+        <PageError
+          message={error}
+          onClick={() => {
```
Comment (bhavya-sinha) [inline src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/PolicyViolations/PolicyViolationsDrawer.tsx:92]: Could be written as onClick={refetch}

**Review by bhavya-sinha — DISMISSED** [review]

File: src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/PolicyViolations/PolicyViolationsDrawer.tsx (hunk 3 above)
Comment (udayvunnam) [reply src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/PolicyViolations/PolicyViolationsDrawer.tsx:92]: onclick and refetch has different syntax with react query.

**Review by PratyushGar — APPROVED** [review]

**Pull request merged**